### PR TITLE
[stable/home-assistant] Add OWNERS file

### PR DESCRIPTION
**Pull request by billimek**
**What this PR does / why we need it**: Just adding an OWNERS file..

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: Adds OWNERS

### CODE REVIEW

File: stable/home-assistant/templates/NOTES.txt [hunk 1]
```diff
@@ -15,5 +15,5 @@
 {{- else if contains "ClusterIP" .Values.service.type }}
   export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "home-assistant.name" . }},release={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
   echo "Visit http://127.0.0.1:8080 to use your application"
-  kubectl port-forward $POD_NAME 8080:80
+  kubectl port-forward $POD_NAME 8080:8123
```
Comment (davidkarlsen) [inline stable/home-assistant/templates/NOTES.txt:18]: tip: this can be done easier, like in this PR: https://github.com/helm/charts/pull/7037/files

**Comment by billimek** [comment]
Now seeing issues with this deploying on a new 1.11 cluster I built.

Going to close this to remediate that issue as well as to make the optional configurator a sidecar instead of a separate deployment

**Pull request closed without merging**